app/views/text.scala.txt

```diff
@@ -0,0 +1,2 @@
+@(text: String)
```

Comment:
I get what you were trying to do here, but you can pass strings directly in the `Ok` (or similar) method.
You can delete this file.